mappings/net/minecraft/world/BlockHelper.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5459 net/minecraft/world/BlockHelper
```

Comment:
If this involves no world instance and is block pos/vec3i handling then should be moved to math.

Reply:
Yes there is nothing world related here. Move it to math and this has nothing to do with `block` as well.